vnext/Microsoft.ReactNative/GlyphViewManager.cpp

```diff
@@ -1,13 +1,13 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 //
-// IconViewManager is a temporary implementation of render Icon from Font
+// GlyphViewManager is a temporary implementation of render Icon from Font
```

Comment:
Is the "temporary implementation" part of this comment still accurate? I'd suggest removing this text or including a link to a tracking issue.

Reply:
Also wondering about "support final impl pending polyester design". (Apologies for the comments on aspects you haven't changed, but I'm hoping this is a good opportunity to tidy up.)

Reply:
I think we can just delete that comment as I don't expect further changes to Glyph's design  :)